jerry-core/lit/lit-magic-strings.c

```diff
@@ -228,7 +228,7 @@ lit_is_utf8_string_pair_magic (const lit_utf8_byte_t *string1_p, /**< first utf-
   while (first <= last)
   {
     int middle = ((first + last) / 2); /**< mid point of search */
-    const lit_utf8_byte_t *middle_string_p = lit_get_magic_string_utf8 (middle);
+    const lit_utf8_byte_t *middle_string_p = lit_get_magic_string_utf8 ((lit_magic_string_id_t) middle);
 
```

Comment:
Sure that a cast is the right fix here? Casting should only be a last resort if there are no other options :)